fast64_internal/oot/oot_skeleton.py

```diff
@@ -12,6 +12,36 @@
 from ..f3d.f3d_material import TextureProperty, tmemUsageUI
 from .oot_f3d_writer import *
 
+
+class OOTSkeletonExportSettings(bpy.types.PropertyGroup):
+    name: bpy.props.StringProperty(name="Skeleton Name", default="gGerudoRedSkel")
+    folder: bpy.props.StringProperty(name="Skeleton Folder", default="object_geldb")
+    customPath: bpy.props.StringProperty(name="Custom Skeleton Path", subtype="FILE_PATH")
+    isCustom: bpy.props.BoolProperty(name="Use Custom Path")
+    removeVanillaData: bpy.props.BoolProperty(name="Replace Vanilla Headers On Export", default=True)
```

Comment:
```suggestion
    removeVanillaData: bpy.props.BoolProperty(name="Replace Vanilla Skeletons On Export", default=True)
```
I assume by "header" you meant "skeleton header" ? I was confused at first with header = .h file

Reply:
Yeah, thats what I meant.